core/templates/dev/head/forms/formBuilder.js

```diff
@@ -723,6 +723,7 @@ oppia.config(['$provide', function($provide) {
       var _openCustomizationModal = function(
           customizationArgSpecs, attrsCustomizationArgsDict, onSubmitCallback,
           onDismissCallback, refocusFn) {
+        document.execCommand('enableObjectResizing', false, false);
```

Comment:
This seems good to me. Only one suggestion: could you change document --> $document and include $document as a dependency? That way, all services that are 'external' to this decorator are explicitly imported.